### PR TITLE
LeetCode:25 Reverse Nodes in k-Group #510 issue

### PR DESCRIPTION
**Pull request by dontwike**
The issue has competed. Please review and let me know if anything needs to be corrected.

### CODE REVIEW

File: JAVA/leetcode solutions (searching)/LC_25_ReverseNodesInkGroup.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
+package LeetCode.LinkList;
```
Comment (Saikat2407) [inline JAVA/leetcode solutions (searching)/LC_25_ReverseNodesInkGroup.java:1]: first line should be question link

**Review by Saikat2407 — CHANGES_REQUESTED** [review]
make the required changes

File: JAVA/leetcode solutions (searching)/LC_25_ReverseNodesInkGroup.java (hunk 1 above)
Comment (dontwike) [reply JAVA/leetcode solutions (searching)/LC_25_ReverseNodesInkGroup.java]: Resolved Please review it once more

**Comment by dontwike** [comment]
I have resolved the issue

**Pull request closed without merging**